packages/asset-service/src/generateAssetData/ethTokens/uniV2.ts

```diff
@@ -0,0 +1,32 @@
+import { AssetNamespace, toAssetId, toChainId } from '@shapeshiftoss/caip'
+import { AssetDataSource, ChainTypes, NetworkTypes, TokenAsset } from '@shapeshiftoss/types'
+
+export const getUniV2Token = (): TokenAsset[] => {
```

Comment:
Better name might be `getUniswapV2TokenPairs` or `getUniswapV2Pools` as this is a bit confusing with the already confusing and somewhat misnamed (imo) `getUniswapTokens`

Reply:
Agree, this should probably be `getUniswapV2Pools`.

We should absolutely make this dynamic, though not one for this PR.